src/back/msl/writer.rs

```diff
@@ -4103,6 +4116,17 @@ impl<W: Write> Writer<W> {
         Ok(info)
     }
 
+    fn write_polyfills(&mut self) -> Result<(), Error> {
+        writeln!(self.out)?;
+        writeln!(
```

Comment:
thought(non-blocking): It'd be nice to avoid two writes and roll the implicit `'\n'` into the next call. I'm not sure what other chunks of the codebase do off the top of my head, though.

Reply:
Thanks! While it doesn't seem to be the existing practice, I've done the change to avoid the extra call now - let me know if I should revert that.